dotnet/src/SemanticKernel/Configuration/KernelConfig.cs

```diff
@@ -320,14 +305,14 @@ public BackendConfig GetEmbeddingsBackend(string? label)
 
         throw new KernelException(
             KernelException.ErrorCodes.BackendNotFound,
-            $"Embeddings backend not found: {label}. No default backend available.");
+            $"Embeddings backend not found with label: {label} and no default embeddings backend is available.");
     }
 
     /// <summary>
     /// Get all completion backends.
     /// </summary>
     /// <returns>IEnumerable of all completion backends in the kernel configuration.</returns>
-    public IEnumerable<BackendConfig> GetAllCompletionBackends()
+    public IEnumerable<IBackendConfig> GetAllCompletionBackends()
```

Comment:
Along the same lines... this will now return a list of IBackedConfigs which contain only a string and no actionable methods. Consider: we could either promote some of the backend functionality into the IBackendConfig, or just change this to GetAllCompletionBackendLabels and return a collection of strings.